Fix:

stat.c:127:9: error: implicitly declaring library function 'strcmp' with type 'int (const char *, const char *)' [-Werror,-Wimplicit-function-declaration]
--- stat.c.orig	2002-10-19 00:04:26.000000000 -0500
+++ stat.c	2021-01-30 23:16:34.000000000 -0600
@@ -40,6 +40,7 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 #include <unistd.h>
 #include <signal.h>
 #include <math.h>
